tensorflow/python/kernel_tests/sparse_ops_test.py

```diff
@@ -510,10 +510,86 @@ def testNoEmptyRows(self):
       self.assertAllEqual(empty_row_indicator_out, np.zeros(2).astype(np.bool))
 
 
+class SparseReduceMaxTest(test_util.TensorFlowTestCase):
```

Comment:
Can you refactor the existing `SparseReduceSumTest` to plumb through an extra bool argument, such as `do_sum == True / False`, so that we can share most of the existing test logic?